### PR TITLE
source/user-manual/capabilities: update CIS-CAT documents to support both v3 and v4

### PR DESCRIPTION
**Pull request by hustliyilin**
Fix https://github.com/wazuh/wazuh/pull/12582

Signed-off-by: YiLin.Li <YiLin.Li@linux.alibaba.com>

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 
- [ ] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

## Note to the reviewer

This PR includes changes to the `redirect.js` script that need to be included in all production branches.

### CODE REVIEW

**Review by vikman90 — CHANGES_REQUESTED** [review]
@hustliyilin Thank you so much for this PR. 

I've made some suggestions for minor changes. Please review them and apply them if you agree.

Cheers!

**Comment by hustliyilin** [comment]
> @hustliyilin Thank you so much for this PR.
> 
> I've made some suggestions for minor changes. Please review them and apply them if you agree.
> 
> Cheers!

Hi @vikman90 ,

Thank you very much. I already modified the codes according to your review.

**Review by vikman90 — APPROVED** [review]
Thank you @hustliyilin.
We're just awaiting another review by @s-ocando, then we will merge both PRs!

Thank you very much for your contribution!
Best regards.

File: source/user-manual/reference/ossec-conf/wodle-ciscat.rst [hunk 1]
```diff
@@ -27,22 +27,25 @@ Main options
...
++======================+============================================================+
+| `disabled`_          | yes, no                                                    |
++----------------------+------------------------------------------------------------+
+| `timeout`_           | A positive number (seconds)                                |
++----------------------+------------------------------------------------------------+
+| `java_path`_         | Any valid path                                             |
++----------------------+------------------------------------------------------------+
+| `ciscat_path`_       | CIS-CAT.sh, CIS-CAT.BAT, Assessor-CLI.sh, Assessor-CLI.bat |
++----------------------+------------------------------------------------------------+
+| `ciscat_binary`_     | Any valid path                                             |
```
Comment (vikman90) [inline source/user-manual/reference/ossec-conf/wodle-ciscat.rst:45]: Seems like if the allowed values column was switched.

```suggestion
| `ciscat_path`_       | Any valid path                                             |
+----------------------+------------------------------------------------------------+
| `ciscat_binary`_     | CIS-CAT.sh, CIS-CAT.BAT, Assessor-CLI.sh, Assessor-CLI.bat |
```

**Review by vikman90 — CHANGES_REQUESTED** [review]
Hi @hustliyilin, we have this feedback from @s-ocando. Please take a look on that and fix it if you agree.

Best regards.

**Comment by hustliyilin** [comment]
> Hi @hustliyilin, we have this feedback from @s-ocando. Please take a look on that and fix it if you agree.
> 
> Best regards.

Hi @vikman90 ,

Thank you very much. I already modified the codes according to your review and rebased to the newest base branch to fix the conflicts.

**Review by vikman90 — APPROVED** [review]
Looks good to me!

**Comment by hustliyilin** [comment]
> Looks good to me!

Thanks  very much @vikman90 ! Cloud you review it again @s-ocando ? Thanks a lot.

**Review by s-ocando — APPROVED** [review]
LGTM. Thank you very much @hustliyilin for your contribution!

**Pull request merged**